packages/next/build/webpack-config.ts

```diff
@@ -7,6 +7,7 @@ import semver from 'next/dist/compiled/semver'
 import { isWebpack5, webpack } from 'next/dist/compiled/webpack/webpack'
 import type webpack5 from 'webpack5'
 import path, { join as pathJoin, relative as relativePath } from 'path'
+import escapeRegExp from 'next/dist/compiled/lodash.escaperegexp'
```

Comment:
We already have a compiled dep for this [here](https://github.com/vercel/next.js/tree/canary/packages/next/compiled/escape-string-regexp)

Reply:
Ah, thanks. I'll use that.